### DISCUSSION TITLE
Istio 1.20.1 CRL

### DISCUSSION THREAD

**Discussion by wmashal** [istio/istio#49050] [Q&A]
Hi
In my  istio 1.20.1 gateway SIMPLE tls termination I add the CRL in my secret like the following , but the request not rejected when I send it with the certificate that listed in the ca.crl , what I expect to see ?  

https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#configure-a-mutual-tls-ingress-gateway

Please advice, if there is a clear documentation for that ?

**Comment by gabriel9895** [comment]
Only in mTLS, the server(ingress gateway) verifies the client certificate, and you can append CRL to the server secrets, in order to check if clients revoked. But in the case of SIMPLE tls, only the client verifies the server(ingress gateway) certificate, not the other way.
